internal/app/api/connector.go

```diff
@@ -118,7 +87,19 @@ func listTasks[Config models.ConnectorConfigObject](connectorManager *integratio
 			}
 		}
 
-		err = json.NewEncoder(w).Encode(response)
+		err = json.NewEncoder(w).Encode(sharedapi.BaseResponse[[]listTasksResponseElement]{
+			Data: &data,
+			Cursor: &sharedapi.Cursor[[]listTasksResponseElement]{
+				PageSize: paginationDetails.PageSize,
+				Total: sharedapi.Total{
```

Comment:
Total is optional. Can be here if not expensive to compute.
It's a legacy field which is filled by search service has elasticsearch always returns this value.

Reply:
Fixed